### PR TITLE
start.sh: Use -n instead of ! -z

### PR DESCRIPTION
**Pull request by zSALLAZAR**

### CODE REVIEW

**Comment by zSALLAZAR** [comment]
See commit message for explanation.

**Comment by SOF3** [comment]
There seems to be two irrelevant changes

**Comment by crischutu07** [comment]
Hey, don't forget to double quotes the "$@"
```
$ shellcheck myscript
# You're solved this one
Line 26:
        elif [[ ! -z $(type php 2> /dev/null) ]]; then
                ^-- SC2236 (style): Use -n instead of ! -z.
 
Line 54:
                "$PHP_BINARY" "$POCKETMINE_FILE" $@
                                                 ^-- SC2068 (error): Double quote array expansions to avoid re-splitting elements.
 
Line 61:
        exec "$PHP_BINARY" "$POCKETMINE_FILE" $@
                                              ^-- SC2068 (error): Double quote array expansions to avoid re-splitting elements.

$```

**Comment by dktapps** [comment]
I guess it would be a good idea to have shellcheck in the CI. I don't know why I never thought of that.

File: start.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 DIR="$(cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
-cd "$DIR"
+cd "$DIR" || exit 1
```
Comment (dktapps) [inline start.sh]: I'm all for fail-fast, but this seems pointless. The script isn't going to fail to cd into its own directory.

**Review by dktapps — COMMENTED** [review]
The individual changes look fine, but I'm not a fan of mixing unrelated changes together.

**Comment by zSALLAZAR** [comment]
> The individual changes look fine, but I'm not a fan of mixing unrelated changes together.

Should I split this pull request into two different ones?

**Comment by dktapps** [comment]
If you wouldn't mind.

**Pull request merged**